eta/core/video.py

```diff
@@ -1546,11 +1546,12 @@ def sample_first_frames(arg, k, size=None):
     # Read frames ...
     if isinstance(arg, six.string_types):
         # ... from disk
-        with FFmpegVideoReader(arg, frames="1-%d" % k) as vr:
+        with FFmpegVideoReader(
+            arg, frames=[i for i in range(1, stride*k, stride)]) as vr:
             imgs = [img for img in vr]
     else:
         # ... from tensor
-        imgs = arg[:k]
+        imgs = arg[:k*stride:stride]
```

Comment:
style: one space before and after `*`. Optional `()` for clarity: `(k * stride)`